catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/PenBricksTest.java

```diff
@@ -0,0 +1,115 @@
+
+import org.catrobat.catroid.ProjectManager;
+import org.catrobat.catroid.R;
+import org.catrobat.catroid.content.Project;
+import org.catrobat.catroid.content.Script;
+import org.catrobat.catroid.content.Sprite;
+import org.catrobat.catroid.content.bricks.PenDownBrick;
+import org.catrobat.catroid.content.bricks.PenUpBrick;
+import org.catrobat.catroid.content.bricks.SetPenColorBrick;
+import org.catrobat.catroid.content.bricks.SetPenSizeBrick;
+import org.catrobat.catroid.ui.ScriptActivity;
+import org.catrobat.catroid.uiespresso.content.brick.utils.BrickTestUtils;
+import org.catrobat.catroid.uiespresso.testsuites.Cat;
+import org.catrobat.catroid.uiespresso.testsuites.Level;
+import org.catrobat.catroid.uiespresso.util.actions.CustomActions;
+import org.catrobat.catroid.uiespresso.util.rules.BaseActivityInstrumentationRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+
+import static org.catrobat.catroid.uiespresso.content.brick.utils.BrickDataInteractionWrapper.onBrickAtPosition;
+
+@RunWith(AndroidJUnit4.class)
+public class PenBricksTest {
+	@Rule
+	public BaseActivityInstrumentationRule<ScriptActivity> baseActivityTestRule = new
+			BaseActivityInstrumentationRule<>(ScriptActivity.class, true, false);
+
+	@Before
+	public void setUp() throws Exception {
+		createProject();
+		baseActivityTestRule.launchActivity(null);
+	}
+
+	@Category({Cat.AppUi.class, Level.Smoke.class})
+	@Test
+	public void testPenBricks() {
+		int positionStartedBrick = 0;
+		int positionPenDownBrick = 1;
+		int positionPenUpBrick = 2;
+		int positionPenSizeBrick = 3;
+		int positionPenColorBrick = 4;
+
+		int penSize = 10;
+		int rgbValueRed = 1;
+		int rgbValueGreen = 2;
+		int rgbValueBlue = 3;
+
+		onBrickAtPosition(positionStartedBrick).checkShowsText(R.string.brick_when_started);
+		onBrickAtPosition(positionPenDownBrick).checkShowsText(R.string.brick_pen_down);
+		onBrickAtPosition(positionPenUpBrick).checkShowsText(R.string.brick_pen_up);
+
+		onBrickAtPosition(positionPenSizeBrick).checkShowsText(R.string.brick_pen_size);
+		onBrickAtPosition(positionPenSizeBrick).onFormulaTextField(R.id.brick_set_pen_size_edit_text)
+				.checkShowsNumber(penSize);
+
+		onBrickAtPosition(positionPenColorBrick).onFormulaTextField(R.id.brick_set_pen_color_action_red_edit_text)
+				.checkShowsNumber(rgbValueRed);
+		onBrickAtPosition(positionPenColorBrick).onFormulaTextField(R.id.brick_set_pen_color_action_green_edit_text)
+				.checkShowsNumber(rgbValueGreen);
+		onBrickAtPosition(positionPenColorBrick).onFormulaTextField(R.id.brick_set_pen_color_action_blue_edit_text)
+				.perform(click());
+		onView(withId(R.id.rgb_blue_value)).perform(click());
```

Comment:
please stick to the espresso coding guidelines and have the check and performs in the next line to help make this a bit more readable.